packages/crypto/src/slip10.ts

```diff
@@ -198,14 +202,14 @@ export class Slip10 {
   }
 
   private static isGteN(curve: Slip10Curve, privkey: Uint8Array): boolean {
-    const keyAsNumber = new BN(privkey);
-    return keyAsNumber.gte(this.n(curve));
+    const keyAsNumber: bigint = bytesToUnsignedBigInt(privkey);
```

Comment:
Type can be inferred, right?
```suggestion
    const keyAsNumber = bytesToUnsignedBigInt(privkey);
```

Reply:
Was trying to make the code self-documenting but I guess 'BigInt' is right in the function name there.